File: src/main/java/org/radarbase/appserver/exception/FcmMessageTransmitException.java
package org.radarbase.appserver.exception;

public class FcmMessageTransmitException extends MessageTransmitException {

    private static final long serialVersionUID = -923871442166939L;

    public FcmMessageTransmitException(String message) {
        super(message);
    }
    public FcmMessageTransmitException(String message, Throwable e) {
        super(message, e);
    }
}

File: appserver-legacy/src/main/kotlin/org/radarbase/appserver/exception/FcmMessageTransmitException.kt
package org.radarbase.appserver.exception

import java.io.Serial

@Suppress("unused")
class FcmMessageTransmitException : MessageTransmitException {
    constructor(message: String) : super(message)
    constructor(message: String, e: Throwable) : super(message, e)

    companion object {
        @Serial
        private const val serialVersionUID: Long = -923871442166939L
    }
}
